From 388d0713f3e3294550f998ae0fd17f2cb3fb9168 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Sun, 20 Jun 2021 00:10:01 -0700
Subject: [PATCH] UnpackAll.py: fix compatibility with SCons-4

* as https://pyup.io/changelogs/scons/#4.0.0 shows:
    - Rename internal Warning base class to SConsWarning to avoid any
    possible confusion with Python's own Warning class.

Signed-off-by: Martin Jansa <martin.jansa@lge.com>
---
 build_common/tools/UnpackAll.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build_common/tools/UnpackAll.py b/build_common/tools/UnpackAll.py
index 19d460bf0..c38134940 100644
--- a/build_common/tools/UnpackAll.py
+++ b/build_common/tools/UnpackAll.py
@@ -101,7 +101,7 @@ import SCons.Util
 
 
 # enables Scons warning for this builder
-class UnpackWarning(SCons.Warnings.Warning) :
+class UnpackWarning(SCons.Warnings.SConsWarning) :
     pass
 
 SCons.Warnings.enableWarningClass(UnpackWarning)
